docs/EXTENDED_USAGE.md

```diff
@@ -11,7 +11,7 @@ cookiebar.issetCookie(1);
 cookiebar.issetCookie('ga');
 
 // Displays the cookie bar (restore: Activates the already confirmed cookie checkboxes)
-cookiebar.show(restore: false);
+cookiebar.show({restore: false});
```

Comment:
Here a boolean value is expected instead of an object. But I understand the confusion. Maybe we better adapt the description above:
```js
// Displays the cookie bar. 
// Parameter: [restore] default: false / Activates the already confirmed cookie checkboxes
cookiebar.show(true);
```